src/main/java/core/basesyntax/strategy/BalanceOperation.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.service.FruitTransaction;
+import core.basesyntax.service.ShopService;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```
Let’s fix it for each handler;)